controllers/database_reconciler_helpers.go

```diff
@@ -95,22 +99,28 @@ func (r *DatabaseReconciler) handleDelete(ctx context.Context, database *ndbv1al
 		log.Info("Checking if database instance exists")
 		allDatabases, err := ndb_api.GetAllDatabases(ctx, ndbClient)
 		if err != nil {
-			log.Error(err, "An error occurred while trying to get all databases")
+			errStatement := "An error occurred while trying to get all databases"
+			log.Error(err, errStatement)
+			r.recorder.Event(database, "Warning", "DatabaseLookupError", errStatement+". "+err.Error())
```

Comment:
Nit pick: There's no space before and after the pluses. Might be harder for the user to read.

Reply:
Reformatted the message.